vet.sh

```diff
@@ -49,7 +49,7 @@ if git status --porcelain | read; then
 fi
 
 git ls-files "*.go" | xargs grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" 2>&1 | tee /dev/stderr | (! read)
-git ls-files "*.go" | xargs grep -l "\"unsafe\"" 2>&1 | grep -v '_test.go' | tee /dev/stderr | (! read)
+git ls-files "*.go" | xargs grep -l "\"unsafe\"" 2>&1 | (! grep -v '_test.go') | tee /dev/stderr | (! read)
```

Comment:
Since the exit status is being checked, I think you can remove the `| tee /dev/stderr | (! read)`.